test/integration/full-content/fixtures/core__gallery.html

```diff
@@ -7,7 +7,7 @@
 	</li>
 	<li class="blocks-gallery-item">
 		<figure>
-			<img src="http://google.com/hi.png" alt="title" />
+			<img src="https://cldup.com/cXyG__fTLN.jpg" alt="title" />
```

Comment:
Could you explain this change? Thanks!

Reply:
Oh I see now, never mind. Would be cool if the image moved so it's in our control.

Reply:
Yes, that's true but what reliable source can we consider under our control?
I used cldup here as we are also using it on the demo page so at least we only depend on this domain.